CONTRIBUTING.md

```diff
@@ -29,19 +25,73 @@ for signing our CLA](https://www.citusdata.com/community/CLA)
    brew link openssl --force
    ```
 
-3. Get the code
+3. Get, build and test the code
 
    ```bash
    git clone https://github.com/citusdata/citus.git
+
+   cd citus
+   ./configure
+   make
+   sudo make install
+   cd src/test/regress
+   make check
    ```
 
-4. Build and test
+#### Linux with Apt-Get (Ubuntu, Debian)
+
+1. Install build dependencies
 
    ```bash
+  sudo sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt/ $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+  wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | sudo apt-key add -
+  sudo apt-get update
+
+  apt-get install -y --no-install-recommends \
```

Comment:
You don't need all of these. Probably:

  * `autotools-dev`
  * `build-essential`
  * `flex`
  * `git`
  * `postgresql-server-dev-all` (or maybe just `-9.5`)
  * `libedit-dev`
  * `libpam0g-dev`
  * `libselinux1-dev`
  * `libxslt1-dev`

Can you try with just these and see whether they're sufficient (they should be…)